src/Datadog.Trace.ClrProfiler.Managed/ExtensionMethods/SpanExtensions.cs

```diff
@@ -15,7 +15,7 @@ internal static string GetHttpMethod(this ISpan span)
             string httpUrl)
         {
             span.Type = SpanTypes.Web;
-            span.ResourceName = resourceName;
+            span.ResourceName = resourceName.Trim();
```

Comment:
@lucaspimentel This should catch any trailing space problems

Reply:
Nice. I missed that before.